examples/sdl_platform.nim

```diff
@@ -18,7 +18,7 @@ import bgfx.platform
 import strutils
 
 proc GetTime*(): float64 =
-    return cast[float64](getPerformanceCounter()*1000) / cast[float64](getPerformanceFrequency())
+    return (float64(getPerformanceCounter()) * 1000'f64) / float64(getPerformanceFrequency())
```

Comment:
Could you remove the `* 1000'f64`? It should match the glfw version... I messed that up awhile back.